tools/accuracy_checker/accuracy_checker/postprocessor/README.md

```diff
@@ -68,4 +68,7 @@ Accuracy Checker supports following set of postprocessors:
 * `resize_super_resolution` - resizing super resolution predicted image. Supported representations: `SuperResolutionAnotation`, `SuperResolutionPrediction`.
   * `dst_width` and `dst_height` - destination width and height for resizing respectively. You can also use `size` instead in case when destination sizes are equal.
     If any of these parameters are not specified, gt high resolution image size will be used as default.
+* `resize_style_transfer` - resizing style transfer predicted image. Supported representations: `StyleTransferAnotation`, `StyleTransferPrediction`.
+  * `dst_width` and `dst_height` - destination width and height for resizing respectively. You can also use `size` instead in case when destination sizes are equal.
```

Comment:
You should also describe `size` as a separate item, despite the above description.

Reply:
Fixed in https://github.com/opencv/open_model_zoo/pull/1106/commits/b78761cab1327b54ae9b3770ff81dbc1f0bee8f6